snapcraft/cli/help.py

```diff
@@ -56,7 +57,16 @@ def help_command(topic, devel):
         snapcraft help sources
         snapcraft help go
     """
-    if topic == 'topics':
+    if not topic:
+        click.echo(ctx.parent.get_help())
+        click.echo(dedent("""\
+
+            To get additional help, run:
```

Comment:
This is quite correct, but it’s odd to think of help as something being “run”, rather than looked up. Instead, maybe “For more help, use:”

The same applies to all the following uses of “run:”